CP/Assignment1-STL/Suraj_Singh/pair.cpp

```diff
@@ -0,0 +1,111 @@
+#include <iostream>
+#include <algorithm>
+using namespace std;
+
+
+
+int main(int argc, char const *argv[])
+{
+	pair <int, int> arr[1000];
+	int flag =0;
+	int num;
+	int i;
+	int j;
+	pair <int, int> swap;
+	for(i=0;i<1000;i++)
+	{
+		arr[i].first = 0;
+		arr[i].second = 0;
+	}
+	cin >> num;
+	
+	for (int k = 0; k < num; ++k)
+	{
+		int tmp;
+		cin >> tmp;
+		int el;
+		int el2;
+		int pos;
+
+		if(tmp==1 || tmp == 2 )
+		{
+			cin >> pos;
+		}
+		if(tmp==1)
+		{
+			cin >> el;
+			cin >> el2;
+			//cout >> el;
+			flag = 1;
+		}
+		switch(tmp)
+		{
+			case 1:
+				arr[pos].first = el;
+				arr[pos].second = el2;
+				
+				break;
+
+			case 2:
+
+				if(pos ==1)
```

Comment:
Use inbuilt sort function. Also if first value is same, you have to check for second value. Apply the same logic for pos=2